### ISSUE TITLE
Implement actions 

### ISSUE THREAD

**Issue by turribeach** [FlyingDiver/Indigo-BMW-CD#3]
Implement actions in this plugin to be able to send commands to the car, lock/unlock, etc.

**Comment by FlyingDiver** [comment]
Possible actions:

Honk Horn
Start Climate
Flash Lights
Lock Doors

**Comment by FlyingDiver** [comment]
See also #7

**Comment by FlyingDiver** [comment]
It'll get added eventually.

**Comment by FlyingDiver** [comment]
Well, this has taken a setback.  I started to work on it today and discovered that the Python module I based my code on is out of date, and the original developer has terminated the project.  I tried to make it work based on his last version, and got nowhere.

The only currently maintained module for the BMW Connected Drive I found requires Python 3.  So it looks like this one is going to get the same treatment as MyQ and Slack.  Which will take a little longer.
